react/features/filmstrip/components/web/Filmstrip.js

```diff
@@ -285,17 +394,35 @@ class Filmstrip extends PureComponent <Props> {
      */
     render() {
         const filmstripStyle = { };
-        const { _currentLayout, _disableSelfView, classes, _visible } = this.props;
+        const {
+            _currentLayout,
+            _disableSelfView,
+            _resizableFilmstrip,
+            _verticalFilmstripWidth,
+            _visible,
+            _verticalViewGrid,
+            classes
+        } = this.props;
+        const { isMouseDown } = this.state;
         const tileViewActive = _currentLayout === LAYOUTS.TILE_VIEW;
+        let maxWidth;
 
         switch (_currentLayout) {
         case LAYOUTS.VERTICAL_FILMSTRIP_VIEW:
+            maxWidth = _resizableFilmstrip
+                ? _verticalFilmstripWidth || DEFAULT_FILMSTRIP_WIDTH
+                : interfaceConfig.FILM_STRIP_MAX_HEIGHT || DEFAULT_FILMSTRIP_WIDTH;
+
             // Adding 18px for the 2px margins, 2px borders on the left and right and 5px padding on the left and right.
-            // Also adding 7px for the scrollbar.
-            filmstripStyle.maxWidth = (interfaceConfig.FILM_STRIP_MAX_HEIGHT || 120) + 25;
+            // Also adding 7px for the scrollbar. Also adding 3px for the drag handle.
+            filmstripStyle.maxWidth = maxWidth + 25 + (_resizableFilmstrip ? 3 : 0);
 
             if (!_visible) {
-                filmstripStyle.right = `-${filmstripStyle.maxWidth + 2}px`;
+                if (_resizableFilmstrip) {
+                    filmstripStyle.right = `-${(this._filmstripRef.current?.offsetWidth ?? 0) + 2}px`;
```

Comment:
Don't we have the filmstrip width in redux? Do we actually need to get it from the reference?